Scripts/aws_stack_stop.ps1

```diff
@@ -0,0 +1,78 @@
+## Powershell script to stop the exisiting AWS stack.
+
+param (
+[parameter(Mandatory=$true)]    [string] $lansa_version,
+[parameter(Mandatory=$true)]    [string] $stop
```

Comment:
Use a boolean. This script is running on the Agent. It does not have the restrictions that the Send-SSMCommand has.

Reply:
@robe070 We cannot pass the boolean argument from pipeline to script So it would be passed as a string to the script. So accepting the argument in the string would work only. This is the same problem we are having in nikhil's script as well.

Reply:
Make the stop parameter in the pipeline take a 0 or 1. Then the script can use Boolean.

Reply:
@robe070 Updated the $stop to boolean type and changed the if logic